### ISSUE TITLE
Hope to replace the feed's icon with the favicon of the original site

### ISSUE THREAD

**Issue by tangkaichuan** [yang991178/fluent-reader#169]
**Is your feature request related to a problem? Please describe.**
At present, I have lots of feeds with the same icon which are generated from RSSHub.

**Describe the solution you'd like**
Replace the feed's icon with the favicon of the original site.

**Comment by duetg** [comment]
建议增加读取feed中的<icon>以及<atom:icon>标签以取代仅读取feed来源的favicon。不是所有阅读器都支持<atom:icon>，但<atom:icon>是标准，目前发现Feedbin是能显示<atom:icon>的。

**Comment by ChaosBlades** [comment]
I noticed a lot of RSS feeds I add pull the icon but Phoronix did not. I had to manually go to https://www.phoronix.com/favicon.ico and manually add it. I thought that is how it worked already but I guess not judging by this.

**Comment by juniperrrrr** [comment]
Yes!! Or make it so manually set icons don't get deleted by the service/API when resyncing feeds/groups 🙏
